scripts/a11y/index.js

```diff
@@ -0,0 +1,84 @@
+const fs = require('fs');
```

Comment:
Do we to add this script in terra-core? Should we be checking for accessibility with webdriver.io tests for the markdown component?

Reply:
This file shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.